airflow/__init__.py

```diff
@@ -37,7 +37,8 @@
 from airflow.configuration import conf
 from airflow.exceptions import AirflowException
 from airflow.models import DAG
-
+# Load SQLAlchemy models during package initialization
+from airflow import jobs  # noqa: F401
```

Comment:
This looks _really_ odd (cos this is `airflow`) - `import airflow.jobs` would be less stange looking.

Rather than putting this here though I think it makes a bit more sense to put it in airflow.models

Reply:
I moved it to airflow.models.

Reply:
This caused problems with the order in which modules were loaded, but after moving to the end, the package loads.